[Bug]: Nightly / CI failed - tests/e2e/accuracy/minimax_h3/test_minimax_h3_t2va_similarity.py::test_minimax_h3_t2va_matches_official_reference,Not Found for url: xxx/73372e6cf53e414edd3ab03e357717fb0602e758/assets/t2va.mp4

### Your current environment

<details>
<summary>The output of <code>python collect_env.py</code></summary>

```text
ci env
```

</details>

### Your code version

<details>
<summary>The commit id or version of vllm</summary>

```text
0.26.0
```
</details>
<details>
<summary>The commit id or version of vllm-omni</summary>

```text
900a7f0813d0
```
</details>

### 🐛 Describe the bug

**Buildkite build:** https://buildkite.com/vllm/vllm-omni/builds/13108
**Buildkite step:** [:full_moon: Diffusion X2V · MiniMax H3 T2VA Accuracy Test](https://buildkite.com/vllm/vllm-omni/builds/13108#019fc8cb-55fa-49d6-9f2d-cb097a7a3fe0)

**Failure kind:** pytest FAILED
**Test node:** `tests/e2e/accuracy/minimax_h3/test_minimax_h3_t2va_similarity.py::test_minimax_h3_t2va_matches_official_reference`

**Log reason:**
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://huggingface.co/MiniMaxAI/MiniMax-H3/resolve/73372e6cf53e414edd3ab03e357717fb0602e758/assets/t2va.mp4

**Analysis:**
Category: skip/xfail related. Confirm markers match intended behavior for this version.

**Error log excerpt:**

```text
accuracy_artifact_root = PosixPath('/workspace/build/buildkite/tests/e2e/accuracy/artifacts')



    @hardware_test(res={"cuda": "H100"}, num_cards=4)

    def test_minimax_h3_t2va_matches_official_reference(

        accuracy_artifact_root: Path,

    ) -> None:

        if not torch.cuda.is_available():

            pytest.skip("MiniMax H3 T2VA accuracy test requires CUDA.")

    

        probe_binary("ffmpeg")

        probe_binary("ffprobe")

        output_dir = reset_artifact_dir(accuracy_artifact_root / "minimax_h3_t2va")

>       reference_path = _download_reference_video(output_dir)

                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^



tests/e2e/accuracy/minimax_h3/test_minimax_h3_t2va_similarity.py:141: 

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

tests/e2e/accuracy/minimax_h3/test_minimax_h3_t2va_similarity.py:97: in _download_reference_video

    response.raise_for_status()

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 



self = <Response [404]>



    def raise_for_status(self) -> None:

        """Raises :class:`HTTPError`, if one occurred."""

    

        http_error_msg = ""

        if isinstance(self.reason, bytes):

            # We attempt to decode utf-8 first because some servers

            # choose to localize their reason strings. If the string

            # isn't utf-8, we fall back to iso-8859-1 for all other

            # encodings. (See PR #3538)

            try:

                reason = self.reason.decode("utf-8")

            except UnicodeDecodeError:

                reason = self.reason.decode("iso-8859-1")

        else:

            reason = self.reason

    

        if 400 <= self.status_code < 500:

            http_error_msg = (

                f"{self.status_code} Client Error: {reason} for url: {self.url}"

            )

    

        elif 500 <= self.status_code < 600:

            http_error_msg = (

                f"{self.status_code} Server Error: {reason} for url: {self.url}"

            )

    

        if http_error_msg:

>           raise HTTPError(http_error_msg, response=self)

E           requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://huggingface.co/MiniMaxAI/MiniMax-H3/resolve/73372e6cf53e414edd3ab03e357717fb0602e758/assets/t2va.mp4



/usr/local/lib/python3.12/dist-packages/requests/models.py:1167: HTTPError

---------------------------------------------------------------------- Captured log setup ----------------------------------------------------------------------

INFO     vllm.config.scheduler:scheduler.py:252 Chunked prefill is enabled with max_num_batched_tokens=2048.

INFO     vllm.config.vllm:vllm.py:1109 Asynchronous scheduling is enabled.

INFO     vllm.config.kernel:kernel.py:295 Final IR op priority after setting platform defaults: IrOpPriorityConfig(rms_norm=['native'], fused_add_rms_norm=['native'])
```

---
*Generated from a nightly HTML report. Redact secrets before submitting; complete the checkboxes on GitHub.*

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://vllm-omni.readthedocs.io), which can answer lots of frequently asked questions.

--- a/tests/e2e/accuracy/minimax_h3/test_minimax_h3_t2va_similarity.py
+++ b/tests/e2e/accuracy/minimax_h3/test_minimax_h3_t2va_similarity.py
@@ -28,7 +28,7 @@ pytestmark = [pytest.mark.benchmark, pytest.mark.diffusion, pytest.mark.full_mod
 MODEL_REPO_ID = "MiniMaxAI/MiniMax-H3"
 MODEL_REVISION = "73372e6cf53e414edd3ab03e357717fb0602e758"
 MODEL_ENV_VAR = "VLLM_TEST_MINIMAX_H3_FL2VA_MODEL"
-REFERENCE_VIDEO_URL = f"https://huggingface.co/{MODEL_REPO_ID}/resolve/{MODEL_REVISION}/assets/t2va.mp4"
+REFERENCE_VIDEO_URL = f"https://huggingface.co/{MODEL_REPO_ID}/resolve/main/assets/t2va.mp4"
 
 # This is the official reproducible 768p T2VA prompt associated with
 # ``assets/t2va.mp4`` in the MiniMax H3 repository.

